python/ray/serve/_private/replica_scheduler/pow_2_scheduler.py

```diff
@@ -272,6 +285,8 @@ def update_replicas(self, replicas: List[ReplicaWrapper]):
         self._replica_queue_len_cache.remove_inactive_replicas(
             active_replica_ids=new_replica_id_set
         )
+        # Populate cache for all replicas
+        self._loop.create_task(self._probe_queue_lens(list(self._replicas.values()), 0))
```

Comment:
hm can we do this only for the replicas that were added instead of all?

Reply:
yes! for some reason I thought it would mess with the fault tolerance, but seems like the actor info is stored per-process not per actor handle. changed to only ping new replicas.